### DISCUSSION TITLE
Setup for Ryzen 5 5600x + 3060 Ti

### DISCUSSION THREAD

**Discussion by foaryon** [Rem0o/FanControl.Releases#413] [Q&A]
Hey,

i wanna use this tool for my setup. I have a bequiet case with two integrated fans, the cpu fan and two on the GPU.

Anything to do beside set everything on auto? Where I can see the "curve" which the for example GPU using auto generated?

Is it normal that the GPU fans stop running if you switch out of a game just a for a sec?

Anyone can share a well working setup? :))

**Comment by Rem0o** [comment]
Nothing is "auto-generated" , you must create your fans manually with the big "+" button, then assign those curves to each control at the top.

**Reply by foaryon** [reply]
How do I know which control is which, I have these:
![image](https://user-images.githubusercontent.com/20794218/144493278-0dc449f3-9f07-4f56-a50b-db761f65c9a8.png)

Possible to share a good proper fan curve for CPU/GPU I can work with and tweak it maybe a bit? :)

Thank you for help.
